app/adaptive_tessellation/LoggerDataCollector.h

```diff
@@ -0,0 +1,189 @@
+#pragma once
+
+#include <spdlog/stopwatch.h>
+#include "AdaptiveTessellation.h"
+#include "wmtk/utils/getRSS.h"
+
+namespace adaptive_tessellation {
+
+class LoggerDataCollector
+{
+    class StatisticsObj
+    {
+        double min_ = std::numeric_limits<double>::max();
```

Comment:
not a big deal as we will have to refactor everything in the future after a discussion on style but i far prefer having prefix annotations for member variables than suffix annotations - for non-trivial types 
matrix_.size() is harder to realize as a member than _matrix.size() (or my favorite is m_matrix.size(), which is distinguished from statics called s_matrix

Reply:
I'll keep them the way they are until we discussed naming conventions.